actors/builtin/miner/miner_test.go

```diff
@@ -3925,14 +3952,15 @@ func TestApplyRewards(t *testing.T) {
 		require.Len(t, vestingFunds.Funds, 180)
 
 		// Vested FIL pays out on epochs with expected offset
-		expectedOffset := periodOffset % miner.RewardVestingSpec.Quantization
+		expectedOffset := periodOffset % miner.RewardVestingSpecV6.Quantization
 
 		for i := range vestingFunds.Funds {
 			vf := vestingFunds.Funds[i]
-			require.EqualValues(t, expectedOffset, int64(vf.Epoch)%int64(miner.RewardVestingSpec.Quantization))
+			require.EqualValues(t, expectedOffset, int64(vf.Epoch)%int64(miner.RewardVestingSpecV6.Quantization))
 		}
 
-		assert.Equal(t, amt, st.LockedFunds)
+		lockedAmt := miner.LockedRewardFromRewardV6(amt)
+		assert.Equal(t, lockedAmt, st.LockedFunds)
```

Comment:
Here is where we can have a test for both the old and new versions. Grandfather the old one by setting the network version to the prior one, and add a new one.

Reply:
Actually now that I've reverted the change to initial delay, there's not really anything to grandfather here. This is just an improvement on the old test.